README.md

```diff
@@ -48,11 +48,12 @@ Usage
 Add the following to your `init.el` (after calling `package-initialize`):
 
 ```el
-(when (memq window-system '(mac ns))
+(when (memq window-system '(mac ns x))
   (exec-path-from-shell-initialize))
```

Comment:
Is this really desirable? In my experience, the X session is started inside a user shell invocation, so `exec-path-from-shell` shouldn't be necessary. There's a startup cost associated with `exec-path-from-shell` which is better to avoid if possible.